sdks/python/apache_beam/transforms/periodicsequence_it_test.py

```diff
@@ -0,0 +1,92 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Integration tests for cross-language transform expansion."""
+
+# pytype: skip-file
+
+import time
+import unittest
+
+import pytest
+
+import apache_beam as beam
+from apache_beam import Pipeline
+from apache_beam.options.pipeline_options import StandardOptions
+from apache_beam.testing.test_pipeline import TestPipeline
+from apache_beam.testing.util import assert_that
+from apache_beam.testing.util import is_empty
+from apache_beam.transforms import trigger
+from apache_beam.transforms import window
+from apache_beam.transforms.core import DoFn
+from apache_beam.transforms.periodicsequence import PeriodicSequence
+
+
+@unittest.skipIf(
+    not TestPipeline().get_pipeline_options().view_as(
+        StandardOptions).streaming,
+    "Watermark tests are only valid for streaming jobs.")
+class PeriodicSequenceIT(unittest.TestCase):
+  def setUp(self):
+    self.test_pipeline = TestPipeline(is_integration_test=True)
+    self.runner_name = type(self.test_pipeline.runner).__name__
+    self.project = self.test_pipeline.get_option('project')
+
+  @pytest.mark.it_postcommit
+  @pytest.mark.sickbay_direct
+  @pytest.mark.sickbay_spark
+  @pytest.mark.sickbay_flink
+  @pytest.mark.timeout(
+      900)  # Timeout after 15 minutes to give Dataflow some extra time
```

Comment:
I am a little worried about 15 min timeout. Let's double it to be safe, to prevent flakiness.

Reply:
It generally takes 10, so I'm not too worried, but I bumped it up